qemu/tests/qemu_guest_agent.py

```diff
@@ -2539,8 +2539,8 @@ def gagent_check_freeze_frozen(self, test, params, env):
         try:
             self.gagent.fsfreeze(check_status=False)
         except guest_agent.VAgentCmdError as e:
-            expected = ("The command guest-fsfreeze-freeze has been disabled "
-                        "for this instance")
+            expected = ("Command guest-fsfreeze-freeze has been disabled: "
+                        "the agent is in frozen state")
```

Comment:
Is there other possibility? What about `the agent might be frozen`?

Reply:
That is not as complicated as you think.  This is just a simply situation for testing freeze a frozen fs. If agent can be frozen, there will be no output, the auto-result still can raise error.